src/features/accounts/api/save-webauthn-credential.ts

```diff
@@ -1,9 +1,9 @@
 import { useMutation } from "react-query"
 import { useNetworkContext } from "features/network"
-import { IdStore, Network, WebAuthnIdentity } from "@liftedinit/many-js"
+import { WebAuthnIdentity } from "@liftedinit/many-js"
 
 export function useSaveWebauthnCredential() {
-  const [network] = useNetworkContext()
+  const { command: n } = useNetworkContext()
```

Comment:
Oh this was a little confusing... We were importing the query network just to use it's URL to create a command network? Why would we do that? Maybe we don't apply the IdStore module outside of this context?

Reply:
Yeah I'm not sure why it was made the way it was.

Reply:
In the end I had to revert this specific change because of https://github.com/liftedinit/alberto/pull/147